farm/modeling/predictions.py

```diff
@@ -102,6 +102,7 @@ def _add_answer(self, string):
         and end indices that are stored in the object. """
         if string == "":
             self.answer = "no_answer"
+            self.aggregation_level = "document"
```

Comment:
I think setting the level here is not so useful. I thought no_answers are already doc level and we were missing document level text predictions.

Didnt we say we need to change https://github.com/deepset-ai/FARM/blob/master/farm/modeling/prediction_head.py#L1426
to aggregation level document?

Reply:
You're right. I incorporated the change that we talked about earlier.